### DISCUSSION TITLE
Iterpreting model pathologies with PIT

### DISCUSSION THREAD

**Discussion by dmi3kno** [inlabru-org/inlabru#219] [Q&A]
PIT seems to be a very nice built-in diagnostics tool. Is there a place where I could read about different types of model pathologies based on PIT shape? 

I have been using SBC to calibrate Stan models. [This section](https://hyunjimoon.github.io/SBC/articles/rank_visualizations.html#plot_ecdf-and-plot_ecdf_diff---ecdf-plots) was very helpful in diagnosing the different model problems.

Based on the intuition from SBC and judging from other outputs from the model, this plot indicates that my model is overly confident. Could this mean that my priors are too narrow(?) or there's something wrong with the data (highly influential outliers) or that the mesh is bad?

![pit](https://github.com/inlabru-org/inlabru/assets/13419011/184e11b2-b561-42f1-b165-8710dddcf6bb)

Is there a place I could learn more about the typical pitfalls and how to spot them?

**References:**
Gelman, Andrew, Xiao-Li Meng, and Hal Stern. 1996. “Posterior Predictive Assessment of Model Fitness Via Realized Discrepancies.” Statistica Sinica 6 (4): 733–60. https://www.jstor.org/stable/24306036.

Held, L., B. Schrödle, and H. Rue. 2010. “Posterior and Cross-validatory Predictive Checks: A Comparison of MCMC and INLA.” In Held, L; Schrödle, B; Rue, H (2010). Posterior and Cross-validatory Predictive Checks: A Comparison of MCMC and INLA. In: Kneib, T; Tutz, G. Statistical Modelling and Regression Structures - Festschrift in Honour of Ludwig Fahrmeir. Berlin: Physica-Verlag (Springer), 91-110., edited by T. Kneib and G. Tutz, 91–110. Berlin: Physica-Verlag (Springer). https://doi.org/10.1007/978-3-7908-2413-1.

Marshall, E. C., and D. J. Spiegelhalter. 2003. “Approximate Cross-Validatory Predictive Checks in Disease Mapping Models.” Statistics in Medicine 22 (10): 1649–60. https://doi.org/10.1002/sim.1403.

**Comment by tmeeha** [comment]
Here is some info on how to interpret PIT: Czado, C., Gneiting, T. and Held, L., 2009. Predictive model assessment for count data. Biometrics, 65(4), pp.1254-1261.
________________________________
From: Dmytro Perepolkin ***@***.***>
Sent: Monday, November 13, 2023 1:01 AM
To: inlabru-org/inlabru ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [inlabru-org/inlabru] Iterpreting model pathologies with PIT (Discussion #219)


PIT seems to be a very nice built-in diagnostics tool. Is there a place where I could read about different types of model pathologies based on PIT shape?

I have been using SBC to calibrate Stan models. This section<https://hyunjimoon.github.io/SBC/articles/rank_visualizations.html#plot_ecdf-and-plot_ecdf_diff---ecdf-plots> was very helpful in diagnosing the different model problems.

Based on the intuition from SBC and judging from other outputs from the model, this plot indicates that my model is overly confident. Could this mean that my priors are too narrow(?) or there's something wrong with the data (highly influential outliers) or that the mesh is bad?

[pit]<https://user-images.githubusercontent.com/13419011/282395042-184e11b2-b561-42f1-b165-8710dddcf6bb.png>

Is there a place I could learn more about the typical pitfalls and how to spot them?
